test/docker/docker-compose.yaml

```diff
@@ -16,32 +16,52 @@ services:
     image: confluentinc/cp-kafka
     depends_on:
       - zookeeper
+    ports:
+      - 29092:29092
```

Comment:
it would be good if you could expose 9092, not 29092, because then you can also test without docker by just running kafka locally, which is how i like to do it.